# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/test/navigation/test_bug386782.html
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae docshell/test/navigation/test_bug386782.html
--- a/docshell/test/navigation/test_bug386782.html	Tue Jun 17 00:10:27 2025 +0000
+++ b/docshell/test/navigation/test_bug386782.html	Tue Jun 17 00:37:22 2025 +0000
@@ -17,8 +17,6 @@
     // designMode document is in a frame inside the window, as this means
     // the editable region is not in the root docshell (a less complicated case).
 
-    const kNewWhiteSpaceNormalizerEnabled =
-      SpecialPowers.getBoolPref("editor.white_space_normalization.blink_compatible");
     var gTests = [
       {
         // <html><body><p>designModeDocument</p></body></html>
@@ -34,12 +32,8 @@
         url: "file_bug386782_contenteditable.html",
         name: "contentEditableNavigate",
         expectedBodyBeforeEdit: "<p>contentEditable</p>",
-        expectedBodyAfterEdit: kNewWhiteSpaceNormalizerEnabled
-          ? "EDITED&nbsp;<p>contentEditable</p>"
-          : "EDITED <br><p>contentEditable</p>",
-        expectedBodyAfterSecondEdit: kNewWhiteSpaceNormalizerEnabled
-          ? "EDITED TWICE&nbsp;<p>contentEditable</p>"
-          : "EDITED TWICE <br><p>contentEditable</p>",
+        expectedBodyAfterEdit: "EDITED&nbsp;<p>contentEditable</p>",
+        expectedBodyAfterSecondEdit: "EDITED TWICE&nbsp;<p>contentEditable</p>",
       },
     ];
 